hack/update-codegen.sh

```diff
@@ -88,3 +88,7 @@ ${GOPATH}/bin/deepcopy-gen \
 
 # Make sure our dependencies are up-to-date
 ${REPO_ROOT_DIR}/hack/update-deps.sh
+
+# Apply Patches
+echo "Applying patches"
+git apply ${REPO_ROOT_DIR}/hack/patches/*.patch
```

Comment:
this should be in `update-deps.sh` and we should list each patch explicitly with a `TODO(issue tracking removal)`

Please include the version of K8s containing this patch in the issue and/or comment.

Reply:
Moved to update-deps.sh and added a README.MD with patch removal information